service/src/main/java/bio/terra/cbas/service/RunSetsService.java

```diff
@@ -65,6 +68,8 @@ public class RunSetsService {
   private final UuidSource uuidSource;
   private final ObjectMapper objectMapper;
   private final CbasContextConfiguration cbasContextConfiguration;
+  private final BardService bardService;
+  private final BardServerConfiguration bardServerConfiguration;
```

Comment:
It feels like a little bit of a code smell that the configuration is being brought in and analyzed here instead of in the `BardService`. I see the usage is for an "is it enabled" check, could we push that check down into the BardService itself to simplify the interface here - and not have to remember to do it if we use the bard service somewhere else, too?

Reply:
That's a good point! I originally had it here so that I would not have to fetch the github method details unnecessarily but that is no longer an issue